recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -96,37 +72,56 @@ def configure(self):
             if self.settings.os == "Linux":
                 self.options["ruy"].shared = True
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
-
-    def build_requirements(self):
-        self.build_requires("ninja/1.10.2")
-
     def requirements(self):
-        self.requires("abseil/20210324.2")
+        self.requires("abseil/20211102.0")
         self.requires("eigen/3.4.0")
         self.requires("farmhash/cci.20190513")
         self.requires("fft/cci.20061228")
-        self.requires("flatbuffers/2.0.0")
+        self.requires("flatbuffers/2.0.5")
         self.requires("gemmlowp/cci.20210928")
         if self.settings.arch in ("x86", "x86_64"):
             self.requires("intel-neon2sse/cci.20210225")
         self.requires("ruy/cci.20210622")
         if self.options.with_xnnpack:
-            self.requires("xnnpack/cci.20211026")
-            self.requires("fp16/cci.20200514")
+            self.requires("xnnpack/cci.20211210")
+            self.requires("fp16/cci.20210320")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 14)
+
+        def lazy_lt_semver(v1, v2):
+            lv1 = [int(v) for v in v1.split(".")]
+            lv2 = [int(v) for v in v2.split(".")]
+            min_length = min(len(lv1), len(lv2))
+            return lv1[:min_length] < lv2[:min_length]
+
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if not minimum_version:
+            self.output.warn(f"{self.name} requires C++14. Your compiler is unknown. Assuming it supports C++14.")
+        elif lazy_lt_semver(str(self.settings.compiler.version), minimum_version):
+            raise ConanInvalidConfiguration(f"{self.name} requires C++14, which your compiler does not support.")
+        if self.options.shared:
+            if self.settings.os == "Linux" and not self.options["ruy"].shared:
+                raise ConanInvalidConfiguration(
+                        f"The project {self.name}/{self.version} with shared=True on Linux requires ruy:shared=True")
+
+    def build_requirements(self):
+        self.build_requires("ninja/1.10.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
Won't it fail, if there are no patches?

Reply:
Why do you think it would fail?

Reply:
Sorry, it was wrong line, it should be ok